README.md

```diff
@@ -32,6 +32,129 @@ TODO
           - name: "my-secret-2"
     ```
 
+## Parameters Reference
```

Comment:
It's incorrect for Parameter Reference to be placed between Usage. Please either remove the Usage or move it